docs/integrations/sources/zendesk-support.md

```diff
@@ -65,6 +65,7 @@ The Zendesk Support source connector supports the following streams:
 - [Ticket Metric Events](https://developer.zendesk.com/api-reference/ticketing/tickets/ticket_metric_events/) \(Incremental\)
 - [Ticket Skips](https://developer.zendesk.com/api-reference/ticketing/tickets/ticket_skips/) \(Incremental\)
 - [Users](https://developer.zendesk.com/api-reference/ticketing/ticket-management/incremental_exports/#incremental-user-export) \(Incremental\)
+- [UserFields](https://developer.zendesk.com/api-reference/ticketing/users/user_fields/#list-user-fields) \(Incremental\)
```

Comment:
Incremental support is mentioned here, but I can see only **full refresh** support from code. Which statement is correct?